SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialSender.cpp

```diff
@@ -414,10 +472,32 @@ void USpatialSender::SendRPC(TSharedRef<FPendingRPCParams> Params)
 		// Target object was destroyed before the RPC could be (re)sent
 		return;
 	}
-
 	UObject* TargetObject = Params->TargetObject.Get();
+
+	if (AActor* TargetActor = Cast<AActor>(TargetObject))
```

Comment:
We should also queue RPCs on actor's subobjects.

Reply:
Added handling of actor subobjects.

Reply:
Added GetOuter() for cases, when TargetObject is not an AActor